src/requirements/requirement-json-generator.ts

```diff
@@ -5,18 +5,49 @@ import {
   DecoratedCollegeOrMajorRequirement,
   EligibleCourses,
   RequirementChecker,
+  Course,
 } from './types';
 import sourceRequirements from './data';
 import filteredAllCourses from './filtered-all-courses';
 
+/**
+ * Special (synthetic) courses, as used in AP/IB equivalent courses generation.
+ */
+const specialCourses: Course[] = [
+  {
+    subject: 'CREDITS',
```

Comment:
What special course is CREDITS? Adding a comment to explain or changing the numbers to constants like in Sam's comment could help explain